src/globals.cpp

```diff
@@ -302,6 +303,7 @@ std::string FileNamePathWorld;
 int FileFormatWorld = 0;
 
 bool IsEpisodeIntro = false;
+bool IsHubLevel = false;
```

Comment:
Please remove the old `IsHub()` definition; I think it's later in this file

Reply:
Done, I'll send soon